devel/ci/integration/tests/test_bodhi_tasks.py

```diff
@@ -138,18 +140,17 @@ def find_bug():
         "ipsilon",
         "--bugs",
         bug_id,
+        "--autokarma",
         update_alias,
     ]
-    celery_tasks = bodhi_container.execute(["ls", "/srv/celery-results"])
-    print(list(celery_tasks))
     try:
-        bodhi_container.execute(cmd)
+        result = bodhi_container.execute(cmd)
```

Comment:
I'm just pedantic here: is the result variable used anywhere?

Reply:
It's not, I'll remove the assignation, thanks.